src/tests/multiplexer-integration-test.ts

```diff
@@ -130,4 +130,65 @@ describe('Multiplexer', () => {
 
     DriverFactory.clearRegistrationsForTesting();
   });
+
+  // TODO(sjmiles): probably should be in particles/tests/* because of Multiplexer.js
+  // TODO(sjmiles): skipped because I cannot figure out how to access the output data
```

Comment:
the log below in the test says - unexpected output data. would you like me to patch the PR and help debug?

Reply:
Yes please, I hit a bit of a wall here. Would appreciate any insights.

Reply:
Yes, I would really appreciate an assist, if you can find the cycles.